### ISSUE TITLE
Which data file correspond to table 4 fasttext?

### ISSUE THREAD

**Issue by SHUMKASHUN** [mlfoundations/dclm#9]
Hi, this is really a great work! May I know which data file in the repo corresponds to table 4 fasttext method as there are many files under exp folder. And the positive/negative samples to train the fasttext. Thank you so much for any help.

**Comment by yzou2cmu** [comment]
Same comment here. Thanks for the great work. Are you going to release the fastText OH-2.5 +ELI5 text classifier?

**Comment by Vaishaal** [comment]
the classifier is available here: https://huggingface.co/mlfoundations/fasttext-oh-eli5

**Comment by yzou2cmu** [comment]
great, thanks a lot!

**Comment by SHUMKASHUN** [comment]
Thanks a lot! Btw, would we be able to access those files under s3 in your exp_data (e.g. filtered data), it seems currently the access is denied. Thank you.

**Comment by Vaishaal** [comment]
If you can tell me which files you want, I can move them to huggingface, we unfortunately cannot make the bucket public.

**Comment by yuzc19** [comment]
@Vaishaal Is it possible to share the script for training this classifier?

**Comment by XueDx** [comment]
> If you can tell me which files you want, I can move them to huggingface, we unfortunately cannot make the bucket public.

Could you please release all classifiers in table 5 for further study?
